src/R/Editor/Impl/Data/WorkspaceVariableProvider.cs

```diff
@@ -85,7 +85,7 @@ internal sealed class WorkspaceVariableProvider : RSessionChangeWatcher, IVariab
                 }
 
                 // May be a package object line mtcars$
-                variableName = TrimToTrailingSelector(variableName);
+                variableName = TrimToFirstSelector(variableName);
```

Comment:
`exists()` won't handle nested members anyway, so we'll have to come up with something else.

So the problem is that `rtvs:::describe_children` doesn't do try/catch around `eval` for the expression that it's given. Technically that's by design, so that it would bubble the error up to the caller - but it also prints it as a side effect, apparently...

One thing that you can do with existing code is do `TryEvaluateAndDescribeAsync` on the expression first (to minimize overhead, pass `REvaluationResultProperties.None`), and check if result is `IRErrorInfo`. If it is, then getting children would fail. If not, then you do `DescribeChildrenAsync` - that one shouldn't throw if the expression itself is valid, it does handle errors when trying to get children. 

For the future, I think it would be good to redesign this API in such a way that `EvaluateAndDescribeAsync` and `DescribeChildrenAsync` are merged - basically, `Children` should be a property in `REvaluationResultProperties`, and if you specify it, then the returned `IRValueInfo` has a filled `Children` collection. Then you could do both of these things in a single operation.